docs/context/ref/Chart.md

```diff
@@ -17,7 +17,7 @@ In addition to handling requests to plot charts, a charting application may use
 
 ## Schema
 
-<https://fdc3.finos.org/schemas/next/context/chart.schema.json> ([github](https://github.com/finos/FDC3/tree/main/schemas/context/chart.schema.json))
+[https://fdc3.finos.org/schemas/next/context/chart.schema.json](https://fdc3.finos.org/schemas/next/context/chart.schema.json) ([github](https://github.com/finos/FDC3/tree/main/schemas/context/chart.schema.json))
```

Comment:
Was the generator changed and run to fix these or are they all manual? Will they get reverted by accident?

Reply:
yes